library_generation/utils/proto_path_utils.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+#  Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+from library_generation.model.generation_config import GenerationConfig
+
+
+def remove_version_from(proto_path: str) -> str:
+    """
+    Remove the version of a proto_path
+    :param proto_path: versioned proto_path
+    :return: the proto_path without version
+    """
+    version_pattern = "^v[1-9]"
+    index = proto_path.rfind("/")
+    version = proto_path[index + 1 :]
+    if re.match(version_pattern, version):
+        return proto_path[:index]
+    return proto_path
+
+
+def get_proto_paths(config: GenerationConfig) -> dict[str, str]:
```

Comment:
I would move this to `GenerationConfig`class, and/or refactor it further to a function `get_library_name_from_proto_path(proto_path:str)`.

As a general best practice, we should try to reduce the number of utils file and move the logics to the relative model classes if possible

Reply:
Done.